gc/base/MemoryPoolAddressOrderedListBase.hpp

```diff
@@ -48,6 +48,7 @@ class MM_MemoryPoolAddressOrderedListBase : public MM_MemoryPool
 	MM_HeapLinkedFreeHeader ** _referenceHeapFreeList;
 	MM_SweepPoolState* _sweepPoolState;	/**< GC Sweep Pool State */
 	MM_SweepPoolManagerAddressOrderedListBase* _sweepPoolManager;		/**< pointer to SweepPoolManager class */
+	MM_HeapLinkedFreeHeader *_lastFreeEntry;							/**< address of the last free entry in the pool; valid after compact; NOT maintained during allocation */ 
```

Comment:
I can move up. I do not see big difference. It's probably here because I put it first here (since last entry makes sense only for ordered pools), but then I realized that CompactScheme deals with top level MemoryPool (although it is very order aware).